### PR TITLE
APIDoc: Removed non-existent functions and added missing return types

### PR DESCRIPTION
**Pull request by Seadragon91**

### CODE REVIEW

File: Server/Plugins/APIDump/Classes/World.lua [hunk 1]
```diff
@@ -3167,7 +3150,14 @@ function OnAllChunksAvailable()</pre> All return values from the callbacks are i
...
 				},
-				Notes = "Spawns a {{cTNTEntity|primed TNT entity}} at the specified coords, with the given fuse ticks. The entity gets a random speed multiplied by the InitialVelocityCoeff, 1 being the default value.",
+				Returns =
+				{
+					{
+						Name = "EntityID",
+						Type = "number",
+					},
+				},
+				Notes = "Spawns a {{cTNTEntity|primed TNT entity}} at the specified coords, with the given fuse ticks. The entity gets a random speed multiplied by the InitialVelocityCoeff, 1 being the default value. Returns the EntityID of the spawned primed TNT entity , or -1 on failure.",
```
Comment (Seadragon91) [inline Server/Plugins/APIDump/Classes/World.lua]: Just see that the there is no check if the `Initialize` returns false.
[World.cpp#L2287](https://github.com/cuberite/cuberite/blob/74e6f41/src/World.cpp#L2287)

Will check if there are more entities / mobs that are missing a check there.

Comment (Seadragon91) [inline Server/Plugins/APIDump/Classes/World.lua]: Will do that in a own pull request.

Comment (Seadragon91) [inline Server/Plugins/APIDump/Classes/World.lua]: Ops wrong number if spawn failed, it should be 0.

**Comment by Seadragon91** [comment]
Reverted the change to `SpawnPrimedTNT`, will be added in a own pull request.

File: Server/Plugins/APIDump/Classes/World.lua [hunk 2]
```diff
@@ -2850,7 +2827,13 @@ function OnAllChunksAvailable()</pre> All return values from the callbacks are i
...
 					},
 				},
-				Notes = "Opens or closes a trapdoor at the specific coordinates.",
+				Returns =
+				{
+					{
+						Type = "boolean",
+					},
+				},
+				Notes = "Opens or closes a trapdoor at the specific coordinates. Returns true on success, false if there is no trap door or it's already opened or closed.",
```
Comment (madmaxoft) [inline Server/Plugins/APIDump/Classes/World.lua]: `or it's already in the requested state` might be better

Comment (Seadragon91) [inline Server/Plugins/APIDump/Classes/World.lua]: Changed it.
